cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1350,6 +1350,12 @@ declare namespace Deno {
    */
   export function unrefTimer(id: number): void;
 
+  export interface WebSocketUpgrade {
```

Comment:
Some JSDoc?

Reply:
Two other points:

- Interfaces are open ended, so it should just be:

  ```ts
  export interface WebSocketUpgrade {
    stream: WebSocketStream;
  }
  ```

- There is no logic (as far as I can tell) that detects the unstable flags and behaves differently, so is there a benefit to including it in unstable, because runtime behaviour will always be to have it.

Reply:
> There is no logic (as far as I can tell) that detects the unstable flags and behaves differently

Indeed, I am still thinking how to ideally do this

Reply:
Personally, I don't see any issue with adding it to the stabilised API.  It is additive, we are unlikely to remove it without a breaking change to the larger API. There is no changes to behaviour to the existing stabilised API.  Unstable provides no benefit except introducing friction to users.

Reply:
I agree, but also returning an unstable api from a stable api without repercussions at runtime seems subideal to me